test/JDBC/expected/BABEL-3507-vu-prepare.out

```diff
@@ -0,0 +1,148 @@
+-- max values causing overflow in avg (BIGINT,INT) 
```

Comment:
what would happen if one of the value is NULL?

Reply:
If one of the values are NULL , they are not use in avg calculation thus not causing overflow